common/xrdp_client_info.h

```diff
@@ -170,7 +180,7 @@ struct xrdp_client_info
     int mcs_early_capability_flags;
 
     int max_fastpath_frag_bytes;
-    int capture_code;
+    int old_capture_code;
```

Comment:
Is this now just here for offset purposes? If so, suggest we call it something like `to_be_reused` so that it doesn't look like it still has a function. There's probably a better name than that.

Reply:
yes, I think enums are the same size as int but I think compiler can decide.